easybuild/easyconfigs/c/CubeGUI/CubeGUI-4.6-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,59 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2019 Juelich Supercomputing Centre, Germany
+# Authors::   Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Score_minus_P'
+
+name = 'CubeGUI'
+version = '4.6'
+
+homepage = 'https://www.scalasca.org/software/cube-4.x/download.html'
+description = """
+ Cube, which is used as performance report explorer for Scalasca and Score-P,
+ is a generic tool for displaying a multi-dimensional performance space
+ consisting of the dimensions (i) performance metric, (ii) call path, and
+ (iii) system resource. Each dimension can be represented as a tree, where
+ non-leaf nodes of the tree can be collapsed or expanded to achieve the
+ desired level of granularity.
+
+ This module provides the Cube graphical report explorer.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://apps.fz-juelich.de/scalasca/releases/cube/%(version_major_minor)s/dist']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['CubeGUI-4.6_qtlibs_symlink_realpath.patch']
+checksums = [
+    '1871c6736121d94a22314cb5daa8f3cbb978b58bfe54f677c4c9c9693757d0c5',  # cubegui-4.6.tar.gz
+    'ffb6b3cb02ffe6401505fcbe6d71cf4f46ecfe1f93e2ac8eaacf51da1010a5a5',  # CubeGUI-4.6_qtlibs_symlink_realpath.patch
+]
+
+configopts = [
+    "--with-qt=$EBROOTQT5"
+]
```

Comment:
Move this part so that it is after the `dependencies`.

Reply:
Done in the new commit.